libvcx/src/v3/handlers/proof_presentation/prover/prover.rs

```diff
@@ -174,3 +174,259 @@ impl Prover {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
These tests should be moved to `disclosed_proof.rs` as they are calling `Prover::create` (I suppose you were trying to stay consistent with `cleanup/proof` before we reverted the location of its similar tests)